src/core/include/math/hal/intnat/transformnat-impl.h

```diff
@@ -350,7 +350,7 @@ void NumberTheoreticTransformNat<VecType>::ForwardTransformToBitReverseInPlace(c
         (*element)[i + 0] += omegaFactor - (omegaFactor >= (modulus - loVal) ? modulus : 0);
         if (omegaFactor > loVal)
             loVal += modulus;
-        (*element)[i + t] = loVal - omegaFactor;
+        (*element)[i + 1] = loVal - omegaFactor;
```

Comment:
Why this change in Forward NTT? 
Please add a comment to explain the change

Reply:
It's a silent bug. Should be constant 1. Previously worked because t is always 1 by the time this code is executed.